rfcs/InputUnion.md

```diff
@@ -135,7 +135,9 @@ A wide variety of solutions have been explored by the community, and they are ou
 
 ## Requirements
 
-1. **Backwards compatible** Adding a new member type to an Input Union must not break previously valid operations
+1. **Backwards compatible** ─ adding a new member type to an Input Union or doing any non-breaking change to existing member types must not result in breaking change. For example, adding a new optional field to member type or changing a field from non-nullable to nullable must not break previously valid client operations.
```

Comment:
Thinking more about this I think `Backwards compatible` can mean different stuff in this RFC, for example, proposed changes to introspection can be `Backwards compatible` or switching type from `input` to `inputUnion` can also be `Backwards compatible`.

How about renaming it to `No implicit breaking changes` or something similar?

Reply:
Or “doesn’t inhibit schema evolution”?

Reply:
@benjie Sounds good 👍

Reply:
Done